intezer_analyze_cli/utilities.py

```diff
@@ -104,6 +109,23 @@ def is_supported_file(file_path):
     return is_supported
 
 
+def is_eml_file(stream: BinaryIO) -> Tuple[bool, Union[str, None]]:
+    mail_parser = email.parser.BytesParser()
+    received_headers = ['To', 'Received']
+    required_headers = ['From', 'Date']
+    try:
+        parsed_email = mail_parser.parse(stream, headersonly=True)
+        date = parsed_email.get('Date')
+        return all(header in parsed_email for header in required_headers) and any(
```

Comment:
is there a magic string for eml files? is this is the same check we are doing on the backend?

Reply:
@almogch yes

Reply:
The same